pythclient/hermes.py

```diff
@@ -0,0 +1,173 @@
+import asyncio
+from typing import TypedDict
+
+import httpx
+import os
+
+from .price_feeds import Price
+
+HERMES_ENDPOINT_HTTPS = "https://hermes.pyth.network/api/"
+HERMES_ENDPOINT_WSS = "wss://hermes.pyth.network/ws"
+
+
+class PriceFeed(TypedDict):
+    feed_id: str
+    price: Price
+    ema_price: Price
+    vaa: str
```

Comment:
did you follow the v1 api schema? maybe good to implement for v2 instead since that is live already e.g. https://hermes.pyth.network/v2/updates/price/latest?ids%5B%5D=0xbd640cddb72063e2ede34c6a0baf6699759b9837fcb06aa0e2fbcecb9b65fde7&ids[]=0x2b9ab1e972a281585084148ba1389800799bd4be63b957507db1349314e47445